SteamKit2/SteamKit2/SteamKit2.csproj

```diff
@@ -44,14 +42,6 @@
         <Pack>true</Pack>
         <PackagePath>readme.txt</PackagePath>
     </Content>
-    <Content Include="gpl.txt">
-        <Pack>true</Pack>
-        <PackagePath />
-    </Content>
-    <Content Include="lgpl.txt">
-        <Pack>true</Pack>
-        <PackagePath />
-    </Content>
     <Content Include="license.txt">
```

Comment:
We also have this license file, dunno if we want to drop this.

Reply:
I kept it for now because it also includes the copyright line. We don't have that anywhere else in the package for now.